This script would run with the wrong interpreter. As we will take care of
running gtk-update-icon-cache and update-desktop-database, just remove it.
--- a/meson.build	2018-03-23 01:12:10.000000000 +0100
+++ b/meson.build	2018-03-23 01:12:24.000000000 +0100
@@ -149,8 +149,4 @@
 if cc.get_id() != 'msvc'
   subdir('data')
   subdir('po') # FIXME: build xgettext
-
-  meson.add_install_script('meson_post_install.py',
-    '@0@'.format(get_option('with-theme-manager'))
-  )
 endif
